devel/util-linux/Portfile

```diff
@@ -19,7 +19,20 @@ long_description    ${name} contains miscellaneous utility programs \
                     and messages. This port contains ONLY those parts \
                     of ${name} that run on Darwin.
 
+# master_sites        https://www.kernel.org/pub/linux/utils/util-linux/v${branch}/
+# The above site redirects permanently (301) to mirrors.edge.kernel.org
 master_sites        https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v${branch}/
+
+# mirrors.edge.kernel.org doesn't work with some earlier Apple root CAs,
+# so we need to disable the certificate check.  This isn't much of a risk,
+# since the distfile has to pass the checksum/size checks, anyway.
+#
+# When updating the port to a new version, some extra manual checking
+# of the distfile would be desirable, by performing a certificate-validated
+# check and/or checking the PGP signature.  Note that the .sign file
+# applies to the uncompressed tarball in this case.
+fetch.ignore_sslcert    yes
```

Comment:
Is it worth checking for the os version here instead of a blanket ignore?

Reply:
> Is it worth checking for the os version here instead of a blanket ignore?

As I mentioned in the first paragraph of the commit message, any version threshold would be a moving target.  Then there would be a need to revisit it.

None of the other 10 ports (nor the one PortGroup) that use this setting make it conditional on the OS version, so using it unconditionally seems to be the normal practice.